homeassistant/components/modbus/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "modbus",
   "name": "Modbus",
   "documentation": "https://www.home-assistant.io/integrations/modbus",
-  "requirements": ["pymodbus==1.5.2"],
+  "requirements": ["pymodbus==2.3.0", "twisted==20.3.0"],
```

Comment:
It doesn't look like `twisted` is being used.

Reply:
Correct, but if we do not install Pymodbus throws a warning about it missing, which I wanted to remove from the log. If you can accept the warning I will remove the requirement.

I have been mailing with the Pymodbus people, and they told me it is not so easy to solve as I thought, they are waiting for an upcoming python release.

Please advice, what you want me to do.

Reply:
@balloob Seems like the warning is better than installing `twisted` but I'd prefer if you weighed in.

Reply:
Yeah totally, let's get the warning and then pymodbus should be fixed to not throw that warning.

Reply:
@bdraco @balloob thanks for the input, I have removed the dependency on twister. Personally I was in doubt about the best road.